.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```
IsaacSim 5.1.0 supports ubuntu 24.04
https://github.com/worv-ai/CostNav/blob/main/Dockerfile